src/azure-cli/azure/cli/command_modules/monitor/commands.py

```diff
@@ -199,3 +211,20 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_metric_alerts', custom_command_type=alert_custom)
         g.command('show', 'get')
         g.generic_update_command('update', custom_func_name='update_metric_alert', custom_func_type=alert_custom)
+
+    with self.command_group('monitor log-analytics workspace', log_analytics_workspace_sdk, custom_command_type=log_analytics_workspace_custom, client_factory=cf_log_analytics_workspace) as g:
+        g.custom_command('create', 'create_log_analytics_workspace')
+        g.generic_update_command('update', custom_func_name='update_log_analytics_workspace')
+        g.command('show', 'get')
+        g.command('delete', 'delete')
+        g.custom_command('list', 'list_log_analytics_workspace')
+        g.command('list-link-workspaces', 'list_link_targets')
+        g.command('get-schema', 'get_schema')
+        g.command('list-usages', 'list_usages')
+        g.command('list-management-groups', 'list_management_groups')
+        g.command('get-shared-keys', 'get_shared_keys')
```

Comment:
All your commands here which use SDK help messages are not consistent with you self-defined ones. You can see the difference through 'az monitor log-analytics workspace -h`.